packages/web-components/src/components/va-text-input/va-text-input.tsx

```diff
@@ -229,12 +221,14 @@ export class VaTextInput {
       handleBlur,
       uswds,
       success,
-      messageAriaDescribedby
+      messageAriaDescribedby,
     } = this;
     const type = this.getInputType();
     const maxlength = this.getMaxlength();
-    const ariaDescribedbyIds = `${messageAriaDescribedby ? 'input-message' : ''} ${error ? 'input-error-message' : ''}`
-      .trim() || null; // Null so we don't add the attribute if we have an empty string
+    const ariaDescribedbyIds =
+      `${messageAriaDescribedby ? 'input-message' : ''} ${
+        error ? 'input-error-message' : ''
```

Comment:
Are the formatting updates a preference thing or does the team need to align on the prettier and linting settings (and also make sure we have it running locally)? https://github.com/department-of-veterans-affairs/component-library/blob/main/.prettierrc.json

Reply:
I've always considered formatting preferences to be voluntary/arbitrary/optional, personally. I like making use of Prettier in my code, especially since VS Code's Prettier plugin just takes care of it for me. But I don't think it needs to be a required thing.